api/tests/routes/backoffice/operations_test.py

```diff
@@ -182,8 +182,13 @@ class GetEventDetailsTest(GetEndpointHelper):
     endpoint_kwargs = {"special_event_id": 1}
     needed_permission = perm_models.Permissions.READ_SPECIAL_EVENTS
 
-    # authenticated user + user session + special event + responses
-    expected_num_queries = 4
+    # get authenticated user
+    # get user session
+    # get special event
+    # get special event statistiques
+    # get special event responses
+    # count special event responses (for pagination pagination)
```

Comment:
```suggestion
    # count special event responses (for pagination)
```
Sauf si tu es un Skaven, yes-yes